include/zephyr/dt-bindings/pinctrl/emsdp-pinctrl.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2023 Synopsys
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_EMSDP_PINCTRL_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_PINCTRL_EMSDP_PINCTRL_H_
+
+#define EMSDP_CREG_BASE		   (0xf0001000)
```

Comment:
this should be obtained from the dts file, not defined here.

Reply:
This isn't standard peripheral and it isn't likely to be reused on some other platdom - so probably it's not a big issue.

But as we are already providing address in device tree pinctrl entries (`reg = <0xf0001000 0x100>;`) let's use if from DT.

And anyway we shouldn't have EMSDP_CREG_BASE defined in the dt-bindings header.